File: programming-basics-python/02b-conditional-statements-exercise/04_toy_shop.py
travel_price = float(input())
number_puzzles = int(input())
number_dolls = int(input())
number_bears = int(input())
number_minions = int(input())
number_cars = int(input())

number_toys = (
    number_puzzles + number_dolls + number_bears + number_minions + number_cars
)
sum_toys = (
    number_puzzles * 2.6
    + number_dolls * 3
    + number_bears * 4.1
    + number_minions * 8.2
    + number_cars * 2
)

if number_toys >= 50:
    sum_toys = sum_toys - sum_toys * 0.25

sum_net = sum_toys - sum_toys * 0.1
difference = abs(travel_price - sum_net)

if sum_net >= travel_price:
    print(f"Yes! {difference:.02f} lv left.")

else:
    print(f"Not enough money! {difference:.02f} lv needed.")

File: a1-softuni-programming-basics-javascript/02b-conditional-statements-exercise/04-toy-shop.js
function toyShop(input) {
  let priceOfExcursion = Number(input[0]);
  let amountOfPuzzles = Number(input[1]);
  let amountOfDolls = Number(input[2]);
  let amountOfTedyBears = Number(input[3]);
  let amountOfMinions = Number(input[4]);
  let amountOfTrucks = Number(input[5]);

  amountOfToys =
    amountOfPuzzles +
    amountOfDolls +
    amountOfTedyBears +
    amountOfMinions +
    amountOfTrucks;
  totalPriceOfToys =
    amountOfPuzzles * 2.6 +
    amountOfDolls * 3 +
    amountOfTedyBears * 4.1 +
    amountOfMinions * 8.2 +
    amountOfTrucks * 2;

  if (amountOfToys >= 50) {
    discount = totalPriceOfToys * 0.25;
    totalPriceOfToys -= discount;
    totalMoneyEarned = totalPriceOfToys - totalPriceOfToys * 0.1;
  } else {
    totalMoneyEarned = totalPriceOfToys - totalPriceOfToys * 0.1;
  }

  if (totalMoneyEarned >= priceOfExcursion) {
    moneyAbsolute = Math.abs(totalMoneyEarned - priceOfExcursion).toFixed(2);
    console.log(`Yes! ${moneyAbsolute} lv left.`);
  } else {
    moneyAbsolute = Math.abs(totalMoneyEarned - priceOfExcursion).toFixed(2);
    console.log(`Not enough money! ${moneyAbsolute} lv needed.`);
  }
}
